py_mmd_tools/mmd_to_csw_iso.py

```diff
@@ -1,10 +1,9 @@
 """
 Tool for converting metadata from MMD format to ISO format using a specific xslt.
-Author:    Massimo Di Stefano,
-Created:   03.10.2020 (dd.mm.YYYY)
-Copyright: (c) Norwegian Meteorological Institute
-Usage: 
-EXAMPLE: 
+
+ License:
+     This file is part of the S-ENDA-Prototype repository (https://github.com/metno/S-ENDA-Prototype).
```

Comment:
but this is wrong.. It is part of the py-mmd-tools repository, and should be linked to the license here. The same goes for the other changes.

Reply:
Ah! - I simply copied the license example from the link to DoD you posted - without reading - I Ill fix it